theano/tensor/tests/test_basic.py

```diff
@@ -1932,6 +1932,61 @@ def _numpy_true_div(x, y):
                         bad_runtime=dict(bad1=(rand(5, 7), rand(5, 7)),
                                          bad2=(rand(5, 7), rand(8, 3))))
 
+BatchedDotTester = makeTester(
+    name='BatchedDotTester',
+    op=batched_dot,
+    expected=(lambda xs, ys:
```

Comment:
this is hard to read. Can you make a function numpy_batched_dot above BatchedDotTester and reuse it here?